fcl/dunefd/reco/reco_supernova_dune10kt_1x2x6.fcl

```diff
@@ -1,3 +1,34 @@
-#include "standard_reco_dune10kt_1x2x6.fcl"
+#include "standard_reco_dune10kt.fcl"
 
+services: 
+{
+    @table::services
+    @table::dunefd_1x2x6_reco_services
+}
+
+physics.producers:
+{
+    @table::physics.producers
+    wclsmcnfsp: @local::dune10kt_1x2x6_mc_nfsp
+}
+
+services.DetectorPropertiesService.NumberTimeSamples: 6000
+services.DetectorPropertiesService.ReadOutWindowSize: 6000
+
+physics.producers.gaushit.CalDataModuleLabel: "wclsmcnfsp:gauss"
+physics.producers.gaushit.HitFilterAlg.MinPulseHeight: [ 2, 2, 2 ]
```

Comment:
What was the motivation for these changes?

Reply:
this was to reduce the output files produced.  Please see my comments above where I described the changes I made in response to your (and Laura's) comments.